2015/presentations/import_deep_dive/text.md

```diff
@@ -0,0 +1,297 @@
+# Import Deep Dive Detours - Petr Viktorin
+
+My talk, Import Deep Dive, gave you a simplified overview of Python's import
+machinery, enough to understand what's going on and where to look for more
+details.
```

Comment:
I'm pretty sure that even people who attended the talk will not remember necessary details when reading the article few months (or maybe even days) later. I think that some introduction is really necessary and if it is impossible you should state that it is required to watch Europython 2015 or PythonPL 2015 videos from the talk